README.md

```diff
@@ -1,6 +1,8 @@
 # Bacula
 
-[![Puppet Forge](https://img.shields.io/puppetforge/v/zleslie/bacula.svg)]() [![Build Status](https://travis-ci.org/xaque208/puppet-bacula.svg?branch=master)](https://travis-ci.org/xaque208/puppet-bacula)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/bacula.svg)](https://forge.puppetlabs.com/puppet/bacula)
+[![Build Status](https://github.com/voxpupuli/puppet-bacula/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-bacula/actions?query=workflow%3ACI)
+[![Donated by xaque208](https://img.shields.io/badge/donated%20by-xaque208-fb7047.svg)](#authors)
```

Comment:
Not sure if I should use `xaque208` or `zleslie` here?

Reply:
`Donated by xaque208/zleslie` with a link to `xaque208` badge makes sense to me.